src/coreclr/debug/createdump/crashreportwriter.cpp

```diff
@@ -174,9 +248,19 @@ CrashReportWriter::WriteStackFrame(const StackFrame& frame)
         WriteValue32("token", frame.Token());
         WriteValue32("il_offset", frame.ILOffset());
     }
+    IXCLRDataMethodInstance* pMethod = frame.GetMethod();
+    if (pMethod != nullptr)
+    {
+        ArrayHolder<WCHAR> wszUnicodeName = new WCHAR[MAX_LONGPATH + 1];
+        if (SUCCEEDED(pMethod->GetName(0, MAX_LONGPATH, nullptr, wszUnicodeName)))
+        {
+            std::string methodName = FormatString("%S", wszUnicodeName.GetPtr());
+            WriteValue("method_name", methodName.c_str());
+        }
+    }
     if (frame.ModuleAddress() != 0)
     {
-        const ModuleInfo* moduleInfo = m_crashInfo.GetModuleInfoFromBaseAddress(frame.ModuleAddress());
+        ModuleInfo* moduleInfo = (ModuleInfo*)m_crashInfo.GetModuleInfoFromBaseAddress(frame.ModuleAddress());
```

Comment:
I realized how broken this is since making ModuleInfo mutable.  I'm changing the set in crashinfo to ModuleInfo* and the rest of the code to use pointers instead of copying.